adapters/ninthdecimal/usersync.go

```diff
@@ -0,0 +1,12 @@
+package ninthdecimal
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewNinthdecimalSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("ninthdecimal", 61, temp, adapters.SyncTypeIframe)
```

Comment:
Is this the correct GDPR Vendor List id? Looks like it resolves to GumGum. I don't see one listed for NinthDecimal.

Reply:
Correct we are not listed, updated to 0 for now.